.eslintrc.js

```diff
@@ -194,6 +205,115 @@ module.exports = {
       ],
       rules: { 'import/no-unused-modules': 'off' },
     },
+    {
+      //This is temporary. We will remove these as dark theme gets ported
+      files: [
+        './packages/desktop-client/public/index.html',
+        './packages/desktop-client/src/components/BankSyncStatus.*',
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏smart! My idea was to add `// eslint-ignore-next-line` everywhere, but your solution is wayyyy cleaner!